cluster-provision/k8s/provision.sh

```diff
@@ -40,7 +40,7 @@ fi
 
 ../gocli/build/cli provision ${provision_dir} --phases ${PHASES} ${SLIM_MODE}
 
-if [[ $PHASES == $PHASES_DEFAULT ]] || [[ $CHECK_CLUSTER == true ]]; then
+if [[ $PHASES == *"k8s" ]] || [[ $CHECK_CLUSTER == true ]]; then
    if [[ $PHASES == "linux" ]]; then
```

Comment:
I think that this case can never happen. Because the previous line already checks whether `k8s` is a suffix on `PHASES`, so the expression in this line can never be true, right?

```suggestion
if [[ $CHECK_CLUSTER != true ]] || [[ $PHASES == "linux" ]]; then
  echo "skipping cluster check when running linux only phase"
  exit 0
fi

```

Reply:
Yes that is much simpler.